src/entities/mellowLpRouter.ts

```diff
@@ -399,14 +404,18 @@ class MellowLpRouter {
 
       try {
         await this.refreshWalletBalance();
-      } catch (_) {
+      } catch (error) {
+        Sentry.captureException(error);
+        Sentry.captureMessage('Wallet user balance failed to refresh after deposit');
         console.error('Wallet user balance failed to refresh after deposit');
```

Comment:
Why are we not using the `throw new Error('')` pattern here like in other places? @ssbarbee

Reply:
Not sure if this place needs to rethrow the error. Whoever wrote this code considered this not to be the case and ignore the error completely. I am just tracking it with this PR. Perhaps it can be challenged once we have more information thanks to the tracking.